portable/ARMv8M/non_secure/portable/GCC/ARM_CM23/portmacro.h

```diff
@@ -90,7 +90,7 @@ typedef unsigned long										UBaseType_t;
 #define portARCH_NAME										"Cortex-M23"
 #define portSTACK_GROWTH									( -1 )
 #define portTICK_PERIOD_MS									( ( TickType_t ) 1000 / configTICK_RATE_HZ )
-#define portBYTE_ALIGNMENT									8
+#define portBYTE_ALIGNMENT									32

```

Comment:
Is this just for the alignment on the MPU regions?  If so please add a comment saying so.  Likewise for the same change in other files.

Reply:
Yes, it is for alignment of MPU regions. Will add a comment.